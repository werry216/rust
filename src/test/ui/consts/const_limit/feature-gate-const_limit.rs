#![const_limit="1"]
//~^ ERROR the `#[const_limit]` attribute is an experimental feature [E0658]

const CONSTANT: usize = limit();

fn main() {}

const fn limit() -> usize {
    let x = 42;

    x * 42
}
