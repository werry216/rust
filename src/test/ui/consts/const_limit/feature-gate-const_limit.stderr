error[E0658]: the `#[const_limit]` attribute is an experimental feature
  --> $DIR/feature-gate-const_limit.rs:1:1
   |
LL | #![const_limit="1"]
   | ^^^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/67217
   = help: add `#![feature(const_limit)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
